scheduler/src/cook/mesos/api.clj

```diff
@@ -1871,7 +1892,14 @@
                                :description "The groups were returned."}
                           400 {:description "Non-UUID values were passed."}
                           403 {:description "The supplied UUIDs don't correspond to valid groups."}}
-              :handler (read-groups-handler conn task-constraints is-authorized-fn)}}))
+              :handler (groups-action-handler conn task-constraints is-authorized-fn)}
+        :delete
+        {:summary "Kill all jobs within a set of groups"
+         :parameters {:query-params QueryGroupsParams}
```

Comment:
Should we name this something other than `QueryGroupsParams`?

Reply:
Broke this out into a separate schema that doesn't include `detailed`.